keyboards/svalboard/keymaps/keymap_support.h

```diff
@@ -28,6 +28,8 @@ enum my_keycodes {
     SV_RIGHT_DPI_DEC,
     SV_LEFT_SCROLL_TOGGLE,
     SV_RIGHT_SCROLL_TOGGLE,
+    SV_LEFT_SCROLL_HOLD,
```

Comment:
Key codes always go at the end, or they break all existing maps.

Reply:
Done, good catch. I've updated the order in both keymap_support.h and vial.mk.